templates/Default/nas-grp-index.html.twig

```diff
@@ -19,25 +19,23 @@
                         quam molestias corporis quod, ea minima accusamus.</p>
                 </div>
             </div>
+            
+            <div class="p-3 mx-auto">
+                <div>
+                    <h4>Research Groups</h4>
+                </div>
+                <div class="dropdown show">
+                    <a class="btn btn-dark dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Please select a research group
+                    </a>
 
-            {% if is_granted('ROLE_DATA_REPOSITORY_MANAGER') or is_granted('ROLE_RESEARCH_GROUP_DATA') %}
-                <div class="p-3 mx-auto">
-                    <div>
-                        <h4>Research Groups</h4>
-                    </div>
-                    <div class="dropdown show">
-                        <a class="btn btn-dark dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                            Please select a research group
-                        </a>
-
-                        <div class="dropdown-menu scrollable-menu overflow-auto" aria-labelledby="dropdownMenuLink">
-                            {% for researchGroup in researchGroups %}
-                                <a class="dropdown-item" href="{{ path('pelagos_app_ui_researchgroup_about', { id: researchGroup.getId}) }}" target="_blank">{{ researchGroup.name }}</a>
-                            {% endfor %}
-                        </div>
+                    <div class="dropdown-menu scrollable-menu overflow-auto" aria-labelledby="dropdownMenuLink">
+                        {% for researchGroup in researchGroups %}
+                            <a class="dropdown-item" href="{{ path('pelagos_app_ui_researchgroup_about', { id: researchGroup.getId}) }}" target="_blank">{{ researchGroup.name }}</a>
```

Comment:
This may be just me, but I think the target should be _self.

Reply:
We don't want it to open in a new tab?